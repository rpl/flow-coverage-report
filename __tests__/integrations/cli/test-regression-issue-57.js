'use babel';

import path from 'path';

import {
  FIXTURE_PATH,
  runFlowCoverageReport
} from '../../common';

const testProjectDir = path.join(FIXTURE_PATH, 'issue-57');

jasmine.DEFAULT_TIMEOUT_INTERVAL = 10000; // 10 second timeout

test('Fixed #57 - NaN in text report', async () => {
  const {stdout, stderr} = await runFlowCoverageReport([
    '-i', `"src/*.js"`
  ], {cwd: testProjectDir});

  const filteredStdout = stdout.split('\n').filter(line => line.indexOf('src/url.js') >= 0);

  expect({filteredStdout, stderr}).toMatchSnapshot();
});
