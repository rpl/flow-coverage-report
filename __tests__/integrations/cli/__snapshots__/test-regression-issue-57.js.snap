// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Fixed #57 - NaN in text report 1`] = `
Object {
  "filteredStdout": Array [
    "│ src/url.js │       flow │     0 % │     7 │       0 │ 7         │",
  ],
  "stderr": "Flow Coverage 0% is below the required threshold 80%
",
}
`;
